### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Skycember**
- [DEMO LINK](https://Skycember.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://Skycember.github.io/layout_product-cards-en/report/html_report/)

### CODE REVIEW

File: src/Styles/product_card.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+  background: fff;
```
Comment (kamal-tufekcic) [inline src/Styles/product_card.css:9]: #?

**Review by kamal-tufekcic — APPROVED** [review]
Kinda seems like you overcomplicated this and ended up not knowing where exactly the extra paddings and margins are coming from. It looks very close to mockup so I'll give it a pass, but I find it easier to work from outside in, start from page margins and paddings and slowly work your way inwards, it makes managing spacings a lot easier later on, at least for me.

File: src/Styles/page_layout.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  color: var(--card-color-text);
+}
+
+body {
+  width: 200px;
```
Comment (VinnieJ-2k20) [inline src/Styles/page_layout.css:9]: Don't hardcode `body` width.

File: src/Styles/product_card.css (hunk 1 above)
Comment (VinnieJ-2k20) [reply src/Styles/product_card.css:9]: Yep, it should be `#fff`
And also it's default color, so this rule is useless here.

File: src/Styles/product_card.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  width: 166px;
```
Comment (VinnieJ-2k20) [inline src/Styles/product_card.css:20]: Try not to hradcode width everywhere.

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,57 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
```
Comment (VinnieJ-2k20) [inline src/index.html:13]: There can be no element without block.

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,57 @@
...
+
+      <div class="card__button">
+        <a
+        data-qa="hover"
+        class="card__button-link"
+        href="#buy"
+      >
+        BUY
+        </a>
+      </div>
```
Comment (VinnieJ-2k20) [inline src/index.html:54]: Why do you need extra div?
Also, transform text using CSS.
```suggestion
      <div class="card__button">
        <a
          data-qa="hover"
          class="card__button-link"
          href="#buy"
        >
          buy
        </a>
      </div>
```

**Pull request closed without merging**